docs/product/explore/session-replay/mobile/index.mdx

```diff
@@ -0,0 +1,72 @@
+
+- **Console:** TODO. Bruno at add
+
+- **Errors:** See all the errors that occurred in the replay (including backend), with links to the corresponding events and [issue(s)](https://docs.sentry.io/product/issues/issue-details/error-issues/), and the impact these issues have had holistically across all users on your application when you hover over the issue ID.
+
+- **Tags:** Find a complete list of built-in fields and custom tags associated with a replay, such as operating system version and name, device specs, release, and user details.
+
+- **Trace:** Connects all the [trace(s)](https://docs.sentry.io/product/sentry-basics/tracing/distributed-tracing/#traces-transactions-and-spans) that happened during the replay.
+
+## SDKs Supported
+
+Session Replay for mobile is currently available for Android, iOS, and React Native SDKs. Support for [Flutter](https://docs.sentry.io/platforms/flutter/) is being tracked on this [GitHub issue](https://github.com/getsentry/sentry-dart/issues/1193).
+
+- iOS, [Needs link, VERSION]
+- Android, [Needs link, VERSION]
+- React Native, [Needs link, VERSION]
+
+## Frequently Asked Questions
+
+**Q: How do you protect user data?**
+A: Sentry is privacy-focused so protecting user data is always top of mind. That’s why by default, our privacy configuration is very aggressive and redacts all text and images to ensure no sensitive user information is collected. We also provide server-side scrubbing on debugging context to additionally filter on the server in case anything slips by.
+
+Additionally, we offer a self-serve deletion capability of individual replays in the UI.
+
+**Q: What is the performance overhead of the Replay package?**
+A: For most mobile applications, the performance overhead of our client SDK will be imperceptible to end-users. In our own testing, the overhead was not noticeable by end-users. However, this testing was not exhaustive and you may discover the recording overhead may negatively impact your web application performance depending on your application complexity.
+
+To reduce the performance overhead, we only take screenshots when something changes on the screen. Our default frame rate is 1 frame (16ms) per second of the main thread time.
+If you experience any performance degradations after installing Session Replay, please open an issue on Github for your respective SDK: [Android](https://github.com/getsentry/sentry-java/issues/new/choose), [iOS](https://github.com/getsentry/sentry-cocoa/issues/new/choose) and [React Native](https://github.com/getsentry/sentry-react-native/issues/new/choose).
+
+**Q: How much does it cost to use Session Replay for mobile?**
+A: Session Replay for mobile is currently in open beta and can be used **free of charge**. It is our intention that it becomes a paid solution in the future (similar to [Session Replay for web](https://sentry.io/pricing/)). The exact release date and pricing have not yet been determined.
```

Comment:
```suggestion
**How much does it cost to use Session Replay for mobile?**

Session Replay for mobile is currently in open beta and can be used **free of charge**. At some point in the future, it will becomes a paid solution (similar to [Session Replay for web](https://sentry.io/pricing/)). The exact release date and pricing have yet to be determined.
```